build/kiali/Dockerfile

```diff
@@ -0,0 +1,43 @@
+ARG VERSION
+
+FROM quay.io/kiali/kiali:${VERSION} as console
+
+FROM golang:1.14 as builder
+
+ARG VERSION
+
+RUN set -eux; \
+    \
+    apt-get update; \
+    apt-get install git;
+
+RUN git clone --depth=1 -b ${VERSION} https://github.com/kiali/kiali.git /go/src/github.com/kiali/kiali;
+
+WORKDIR /go/src/github.com/kiali/kiali
```

Comment:
If you move `WORKDIR` above this `RUN` you can simply `git clone --depth=1 -b ${VERSION} https://github.com/kiali/kiali.git  .`